nixos/lib/testing/driver.nix

```diff
@@ -175,7 +175,7 @@ in
   };
 
   config = {
-    _module.args.hostPkgs = config.hostPkgs;
+    _module.args.hostPkgs = config.hostPkgs.__splicedPackages;
```

Comment:
@roberth 
Is https://github.com/nixos/nixpkgs/commit/a742767bafecf0644585309b744e24f90cdff207 relevant for the tests, should I copy it too?

Reply:
In theory yes, but might be low impact.